test/mat_utest.cpp

```diff
@@ -27,14 +27,17 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+#include <gtest/gtest.h>
+
 #include "quadiron.h"
 
-void mat_utest1()
-{
-    std::cout << "mat_utest1\n";
+namespace gf = quadiron::gf;
+namespace vec = quadiron::vec;
 
-    quadiron::gf::Prime<uint32_t> gf11(11);
-    quadiron::vec::Matrix<uint32_t> mat(gf11, 3, 3);
+TEST(MatrixTest, Test1) // NOLINT
```

Comment:
@lamphamsy If you have a better name than Test1, I'll be glad to rename it (it was named utest1 before so I kept it like that)

Reply:
Done